### PR TITLE
Set up basic component and render out simple example

### PR DESCRIPTION
**Pull request by josephschmitt**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ module.exports = {
           }]
         ]
       }
+    }, {
+      test: /\.html$/,
```
Comment (onlywei) [inline webpack.config.js]: sweet webpack-fu

File: example/example.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <body>
+    <main>
+      <ngc-omnibox></ngc-omnibox>
+    </main>
+
+    <script src="../dist/ngc-omnibox.js"></script>
+    <script type="text/javascript">
+      var demoApp = angular.module('demoApp', ['ngc.omnibox']);
```
Comment (onlywei) [inline example/example.html]: There's no need to include `var demoApp =` it seems like. Let's not encourage polluting the global scope :)

Comment (josephschmitt) [inline example/example.html]: 👍

Comment (josephschmitt) [inline example/example.html]: I guess I can always do an IIFE later on if we do need a var

**Comment by josephschmitt** [comment]
@drawyan they just published a blogpost saying everyone needs to get on the 2.2 train because it's shipping in the next month

**Pull request merged**